Sources/Datadog/RUMMonitor.swift

```diff
@@ -137,7 +139,7 @@ public class RUMMonitor: DDRUMMonitor {
                 time: dateProvider.currentDate(),
                 identity: viewController,
                 path: path,
-                attributes: aggregate(attributes)
+                attributes: attributes ?? [:]
```

Comment:
we can set default value to `[:]` instead of `nil`, then we won't need to do that in these places

Reply:
Yup, done 👍.